src/components/Landing/index.tsx

```diff
@@ -190,9 +233,20 @@ export default function Landing() {
               />
             </Col>
           </Row>
+          <Row
+            gutter={40}
+            align="middle"
+            style={{ ...statsRowStyles, display: 'none' }}
+          >
+            {statsCol('292', t`Projects powered by Juicebox`)}
+            {statsCol(`$94,137,431`, t`Raised on Juicebox`)}
```

Comment:
**question**: should these numbers be hard coded?

Reply:
Isn't being shown at the moment. Just leaving it there for when we get the subgraph queries done.

Reply:
Ah k cool, in that case, I'd suggest we delete it for now to avoid the risk of it leaking out and rendering unexpectedly. Can always add it later 🤷